README.md

```diff
@@ -115,3 +115,24 @@ df = farsante.pandas_df([person.full_name, person.email, address.city, address.s
 df.to_parquet('./tmp/fake_data.parquet', index=False)
 ```
 
+## h2o data generation
+
+[h2o](https://github.com/h2oai/db-benchmark) is a popular dataset to benchmark data transformation performance. The main.rs file in [h2o-data-rust](h2o-data-rust/) can be used to generate h2o data.
+
+### arguments
+
+flag | description
+:--- | :---
+--n | number of rows
+--k | number of groups
+--nas | percentage of missing values
+--seed | the seed to use
+
+Here is a command to create a 1 million row dataset with 100 groups, 10% missing values, and a seed of 42.
+
+```bash
+cd h2o-data-rust
+cargo run -- --n 1000000 --k 100 --nas 10 --seed 42
+```
```

Comment:
`cargo run` is two times slower, than uding `cargo build --release` and running optimized compiled binary..